### PR TITLE
inspecting our code with flake8 

### PR DESCRIPTION
**Pull request by RawanAbuleil**
admin.py , models.py , tests.py files has been modified

### CODE REVIEW

**Comment by thetaxas2468** [comment]
Well done!

File: .github/workflows/flake8.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    container: fedora:34
+    steps:
+      - name: Install Pipenv and Git
+        run: dnf install -y pipenv git
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Setup environment
+        run: pipenv sync --dev
+      - name: Run Flake8
+        run: pipenv run flake8 --max-line-length 120
```
Comment (yftacherzog) [inline .github/workflows/flake8.yml:16]: See my comment regarding newline characters here: https://github.com/redhat-beyond/beyond08-group3/pull/4#discussion_r1017531170

**Comment by yftacherzog** [comment]
@RawanAbuleil does this PR needs to be rebased on all those other PRs? Might be that we can merge yours earlier, so that some of that python code is introduced already according to flake8 standards, rather than having you fix it here.

**Comment by yftacherzog** [comment]
@RawanAbuleil try making the PR name more descriptive.

File: .github/workflows/flake8.yml [hunk 2]
```diff
@@ -13,4 +13,5 @@ jobs:
       - name: Setup environment
         run: pipenv sync --dev
       - name: Run Flake8
-        run: pipenv run flake8 --max-line-length 120
\ No newline at end of file
+        run: pipenv run flake8 --max-line-length 120
+
```
Comment (nofaralfasi) [inline .github/workflows/flake8.yml:17]: I don't think you needed to create a new commit for that, right?
You could add this change to your last commit using the `git commit --amend` command.

Comment (yftacherzog) [reply .github/workflows/flake8.yml:17]: @RawanAbuleil please refer to some of the discussions started on other PRs and on Slack regarding pushing fixes to an existing PR.

Comment (RawanAbuleil) [reply .github/workflows/flake8.yml:17]: alright , thanks

**Comment by yftacherzog** [comment]
@RawanAbuleil you're again in some sort of merge-commit. Not sure how come. Please consult with the team regarding how to avoid this (the rest of the PRs look ok in that sense at the moment). I'll merge this tomorrow morning only if this is sorted.

**Comment by pazMenachem** [comment]
LGTM!

File: .github/workflows/flake8.yml (hunk 2 above)
Comment (pazMenachem) [reply .github/workflows/flake8.yml]: Missing one new line at the end of the  yml file, other then that, all looks good!

File: .github/workflows/flake8.yml (hunk 1 above)
Comment (pazMenachem) [inline .github/workflows/flake8.yml:16]:
```suggestion
        run: pipenv run flake8 --max-line-length 120
        *new line*
```

**Review by pazMenachem — COMMENTED** [review]
YML file, nothing much.

**Comment by yftacherzog** [comment]
@RawanAbuleil The merge commit issue was not addressed. See the `Files changed` tab and you see it contains stuff which are not part of your PR. The knowledge for avoiding that already exists within the team, so please ask around.

**Comment by thetaxas2468** [comment]
Looks good,but i think README and CONTRIBUTING files should not be there because they are on their own PRS,great work!

**Comment by yftacherzog** [comment]
lgtm. Need to verify that the workflow is working as expected after the PR is merged.

**Pull request merged**